build.gradle

```diff
@@ -136,6 +144,7 @@ publishing {
         node.appendNode('name', 'Elastic JNA Distribution')
         node.appendNode('description',
           'A build of jna which supports all platforms supported by Elasticsearch')
+        node.appendNode('url', 'https://github.com/elastic/jna-build')
```

Comment:
Why is this necessary? I don't think we should use our own repo because we are just repackaging jna. But I was also able to publish this to maven central before without adding it (at least I don't remember modifying the pom manually...).